### PR TITLE
Updated Dutch translations for validator

### PR DESCRIPTION
**Pull request by BartBaaten**
| Q             | A
| ------------- | ---
| Branch?       | 7.2
| Bug fix?      | no
| New feature?  | no
| Deprecations? | no
| Issues        | no
| License       | MIT

Updated some Dutch translations

### CODE REVIEW

**Review by ro0NL — APPROVED** [review]
i notice "it" is translated as "hij" (he), which sounds odd

edit: it's actually inconsistent currently

File: src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf [hunk 1]
```diff
@@ -400,7 +400,7 @@
             </trans-unit>
             <trans-unit id="103">
                 <source>The value of the netmask should be between {{ min }} and {{ max }}.</source>
-                <target>De waarde van de netmask moet zich tussen {{ min }} en {{ max }} bevinden.</target>
+                <target>De waarde van de netmasker moet tussen {{ min }} en {{ max }} liggen.</target>
```
Comment (rosier) [inline src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: Maybe: het netwerkmasker ?

Comment (BartBaaten) [reply src/Symfony/Component/Validator/Resources/translations/validators.nl.xlf]: Absolutely right!

**Comment by BartBaaten** [comment]
> i notice "it" is translated as "hij" (he), which sounds odd
> 
> edit: it's actually inconsistent currently

I changed it to 'het', which also sounds a bit odd but is probably better indeed.

**Comment by darthf1** [comment]
Or "Deze" instead of "Het"? Or, "De waarde is te lang, deze ... " but maybe that's a bit too much of a change.

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @BartBaaten.

**Pull request merged**